securityconfig/elasticsearch.yml.example

```diff
@@ -188,17 +188,14 @@ opendistro_security.audit.type: internal_elasticsearch
 # constructor that takes an org.elasticsearch.common.settings.Settings
 #opendistro_security.cert.intercluster_request_evaluator_class: com.amazon.opendistroforelasticsearch.security.transport.DefaultInterClusterRequestEvaluator
 
-# Allow snapshot restore for normal users
-# By default only requests signed by an admin TLS certificate can do this
-# To enable snapshot restore for normal users set 'opendistro_security.enable_snapshot_restore_privilege: true'
-# The user who wants to restore a snapshot must have the 'cluster:admin/snapshot/restore' privilege and must also have
-# "indices:admin/create" and "indices:data/write/index" for the indices to be restores.
+# By default, normal users can restore snapshots if they have the priviliges 'cluster:admin/snapshot/restore', 
```

Comment:
Previous comment does mention that snapshots can be restored by normal users. I do like the wording of your version more. Thanks for the update.